.github/workflows/linux_simd.yml

```
@@ -162,12 +162,12 @@ jobs:
           ]
           - [
             "without avx512",
-            "-Dallow-noblas=true -Dcpu-dispatch=SSSE3,SSE41,POPCNT,SSE42,AVX,F16C,AVX2,FMA3",
+            "-Dallow-noblas=true -Dcpu-dispatch=max-x86_v4",
             "3.11"
           ]
           - [
             "without avx512/avx2/fma3",
-            "-Dallow-noblas=true -Dcpu-dispatch=SSSE3,SSE41,POPCNT,SSE42,AVX,F16C",
+            "-Dallow-noblas=true -Dcpu-dispatch=max-x86_v3",
```

```suggestion
            "-Dallow-noblas=true -Dcpu-dispatch=max-x86_v3",
```
This test is no longer needed, since there's a CI with baseline only.